reference/customers.v2.yml

```diff
@@ -183,7 +185,7 @@ paths:
                 _authentication:
                   type: object
                   properties: {}
-                  description: 'This can vary depending on the action being taken to update, validate or force a password change. See Update Customer'
+                  description: This can vary depending on the action being taken to update, validate or force a password change. See Update Customer.
```

Comment:
```suggestion
                  description: This can vary depending on the action being taken to update, validate or force a password change. See [Update Customer]().
```
add the link.

Reply:
```suggestion
                  description: This can vary depending on the action being taken to update, validate, or force a password change. See [Customers V3, Update a customer]().
```